mosquitto/docker-compose.yml

```diff
@@ -2,12 +2,11 @@
 version: "3"
 services:
   mosquitto:
-    image: hub.foundries.io/<your factory>/mosquitto:latest
-    tmpfs:
-      - /run
-      - /var/lock
-      - /var/log
+    image: eclipse-mosquitto:1.6.12
+    volumes:
+      - /var/run/secrets/:/mosquitto/secrets
+      - ./run.sh:/run.sh:ro
```

Comment:
don't you need to mount mosquitto.conf? ie
~~~
  - ./mosquitto.conf:/mosquitto/config/mosquitto.conf:ro
~~~